src/gsy_e/models/strategy/forward/__init__.py

```diff
@@ -94,14 +119,122 @@ def event_offer_traded(self, *, market_id, trade):
     def event_bid_traded(self, *, market_id, bid_trade):
         """Method triggered by the MarketEvent.BID_TRADED event."""
 
+    @property
+    def fully_automated_trading(self):
+        """Is the strategy in fully automated trading mode."""
+        return self.enabled and self._create_order_updater
+
     def event_tick(self):
-        self._update_open_orders()
+        if self.enabled:
+            self._update_open_orders()
 
     def event_market_cycle(self):
         self._delete_past_order_updaters()
-        self._post_orders_to_new_markets()
+        if self.fully_automated_trading:
+            self._post_orders_to_new_markets()
 
     @property
     def state(self) -> "StateInterface":
         """Get the state class of the strategy. Needs to be implemented by all strategies"""
         raise NotImplementedError
+
+    def apply_live_event(self, event: Dict):
```

Comment:
I am missing the instance, that is actually calling this method, so the connection between gsy-web and here. Did I miss this?

Reply:
Thanks for the notification, I unfortunately missed committing my changes to the live_events.py file :( 
Having a second look at these changes though, I noticed that I had handled the live events for the forward markets similarly to the other live events - essentially processing them on every market slot. This will not be good for these though, since they involve trading commands. For this purpose, I will also change the event handling specifically for these events, to be processed on every tick and not on every market slot. Let me know if you do not agree, otherwise I will add this fix too. Thanks!

Reply:
Your implementation makes sense. Thanks!